src/parser/parser_MAT.c

```diff
@@ -392,8 +392,12 @@ void MAT_PARSER_load(MAT_Parser* parser, Net* net) {
     BUS_set_name(bus,mat_bus->name);
     BUS_set_v_mag(bus,mat_bus->Vm,0);         // per unit
     BUS_set_v_ang(bus,mat_bus->Va*PI/180.,0); // radians
-    BUS_set_v_max(bus,mat_bus->maxVm);        // per unit
-    BUS_set_v_min(bus,mat_bus->minVm);        // per unit
+    BUS_set_v_max_reg(bus,mat_bus->maxVm);    // per unit
+    BUS_set_v_min_reg(bus,mat_bus->minVm);    // per unit
+    BUS_set_v_max_norm(bus,mat_bus->maxVm);   // per unit
+    BUS_set_v_min_norm(bus,mat_bus->minVm);   // per unit
+    BUS_set_v_max_emer(bus,mat_bus->maxVm);   // per unit
+    BUS_set_v_min_emer(bus,mat_bus->minVm);   // per unit
```

Comment:
matpower files do not have emer,reg limits info. Please change the code here to only set "norm" voltage limits. The other ones should be kept with default values.